ui/cryptomaterial/editor.go

```diff
@@ -135,6 +159,10 @@ func (t *Theme) Editor(editor *widget.Editor, hint string) Editor {
 		},
 		CustomButton: t.Button(""),
 	}
+
+	newEditor.editorCard = Card{Color: newEditor.backgound}
+	newEditor.editorCard.Radius = Radius(8)
```

Comment:
I'm not sure we need an `editorCard` property inside the `Editor` struct. We can always create this when it's needed.

Reply:
Let's have a `cardBackgroundColor` property then. I assume you mean that card background won't always be the editor background, right? The `cardBackgroundColor` can use the editor background as the default value, just as you do here.

Reply:
okay